app/index.tsx

```diff
@@ -11,12 +11,14 @@ import Countdown from "./containers/Countdown";
 const root = document.getElementById("react-root");
 
 const render = () => {
-  ReactDOM.render(
+  if (root) {
```

Comment:
could you leave additional comment here that we're doing it because of the fact that we load whole bundle on each subpage?